src/WebAPI/Controllers/TicketsController.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using WebAPI.Models;
+
+namespace WebAPI.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/[controller]")]
+    [Authorize(Roles = "User")]
+    public class TicketsController : Controller
+    {
+        // GET /tickets
+        [HttpGet]
+        public IEnumerable<TicketModelResponse> Get()
+        {
+            List<TicketModelResponse> tickets = new List<TicketModelResponse>();
+            return tickets;
+        }
+
+        // GET /tickets/{id}
+        [HttpGet("{id:int}")]
+        public IEnumerable<TicketModelResponse> GetById(int id)
+        {
+            List<TicketModelResponse> tickets = new List<TicketModelResponse>();
+            return tickets;
+        }
+
+        // GET /tickets/new-ticket/{place-id} -> get ticket that has been created
+        [HttpGet]
```

Comment:
Something wrong with this method. Looks like it should be deleted. Need to discuss